routes/api.php

```diff
@@ -40,7 +40,7 @@
             Route::get('enroute/all', 'API\StatusController@enroute')->name('api.v0.statuses.enroute');
             Route::get('event/{statusId}', 'API\StatusController@getByEvent')->name('api.v0.statuses.event');
             Route::post('{statusId}/like', 'API\StatusController@createLike')->name('api.v0.statuses.like');
-            Route::delete('{statusId}/like', 'API\StatusController@destroyLike')->name('api.v0.statuses.like');
```

Comment:
does this really work? IIRC laravel tries to match a route name on name and request-method. I'll have to check on this.

Reply:
Okay, seems to work.